packages/terra-functional-testing/src/terra-dev-site/terra-functional-testing/migration-guide/Migration.tool.mdx

```diff
@@ -0,0 +1,68 @@
+# Migration Guide
+
+A guide for migrating from [terra-toolkit's](https://github.com/cerner/terra-toolkit-boneyard) WDIO testing framework to terra-functional-testing.
+
+## Introduction
+
+Over the years [terra-toolkit](https://github.com/cerner/terra-toolkit-boneyard) has continued to grow and expand; overreaching the responsibilities a single package should have. The decision was reached to reorganize the terra-toolkit utilities into packages that encapsulated specific functionality. terra-functional-testing was created to facilitate the WDIO testing automation framework and make necessary upgrades and changes.
+
+## Getting Started
+
+To get started, read through the changes below and become familiar with the the upgraded packages. There were many breaking changes between WebdriverIO v4 and WebdriverIO v6.
```

Comment:
maybe summary our change in syntax, etc ? or move the comment on wdio to it's bullet?